### PR TITLE
Css modules require hook

### PR DESCRIPTION
**Pull request by earnubs**
replace webpack isomorphic tools with require hooks for a little less magic, and allow easier refactoring and testing of the server code.

### CODE REVIEW

File: test/routes/t_api.js [hunk 1]
```diff
@@ -1,94 +1,133 @@
...
-import { agent } from 'supertest';
+import supertest, { agent } from 'supertest';
+import session from 'express-session';
+
+import api from '../../src/server/routes/api';
+import conf from '../../src/server/configure';
+import sessionConfig from '../../src/server/helpers/session';
 
+/**
+ * Universal route captures all, meaning we can't modify the app route stack
```
Comment (earnubs) [inline test/routes/t_api.js]: These tests have some value now in the absence of acceptance tests, but may be considered too fragile once we're testing e2e.

File: src/server/index.js [hunk 2]
```diff
@@ -1,10 +1,21 @@
+const url = require('url');
+const util = require('util');
 require('babel-register');
+require('css-modules-require-hook/preset');
 
-const WebPackIsomorphicTools = require('webpack-isomorphic-tools');
-const basePath = require('path').resolve(__dirname, '../..');
+const conf = require('./configure.js');
+const WEBPACK_DEV_URL = conf.get('SERVER:WEBPACK_DEV_URL') || '';
+require('images-require-hook')('.svg', `${WEBPACK_DEV_URL}/static/icons`);
```
Comment (bartaz) [inline src/server/index.js]: So currently with this line we only have support for importing `.svg` images?
What does the `static/icons` path stands for? Do we have such directory with images? I only recall some svg images that are in components folders along with the code.

Comment (earnubs) [inline src/server/index.js]: As configured, `.svg` only, but adding support for other types is jus a matter of adding the extension to the first require hook argument, and to the file loader test in webpack config.

`static/icons` is where the images require hook will put the files it encounters on build https://github.com/ptshih/images-require-hook#parameters

**Comment by earnubs** [comment]
[Refactor express server](https://trello.com/c/chxSMtVe/108-refactor-express-server)

**Pull request merged**